src/functions/formatBlockData.js

```diff
@@ -34,7 +33,12 @@ export default async function formatBlockData(blocks) {
 
       const innerBlocksFormatted = await formatBlockData(innerBlocks);
 
-      return { name, attributes, innerBlocks: innerBlocksFormatted };
+      return {
+        name,
+        attributes,
+        content: saveContent,
+        innerBlocks: innerBlocksFormatted,
+      };
     })
   );
 }
```

Comment:
Do we still need changes in here? 

I went to a conclusion that we don't need this function at all.